spec/support/fake_github_webhook.rb

```diff
@@ -0,0 +1,45 @@
+class FakeGithubWebhook
```

Comment:
This is actually a general-purpose fake webhook class, so you should name it something more general thatn FakeGithubWebhook - maybe just FakeWebhook